libbeat/cmd/instance/beat.go

```diff
@@ -695,7 +697,8 @@ func (b *Beat) configure(settings Settings) error {
 
 func (b *Beat) loadMeta(metaPath string) error {
 	type meta struct {
-		UUID uuid.UUID `json:"uuid"`
+		UUID       uuid.UUID `json:"uuid"`
+		FirstStart time.Time `json:"first_start"`
```

Comment:
this will need some testing. the stdlib JSON encoder/decoder might not play well with that type.

Reply:
In my manual tests, it worked like a charm. What might be the problem?

Reply:
My concern would be that timestamp serialization and parsing is not supported. If it works +1. We have some tests for loadMeta I think. Still would be nice to enhance the existing tests.